js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -617,6 +617,7 @@ const DropDownEditor = TextBox.inherit({
             fullScreen: false,
             contentTemplate: null,
             _wrapperClassExternal: DROP_DOWN_EDITOR_OVERLAY,
+            _ignoreElementAttrDeprecation: true,
```

Comment:
Seems like we should specify `_ignorePreventScrollEventsDeprecation` here

Reply:
https://github.com/DevExpress/DevExtreme/pull/24793/commits/8938889ce11b97ce41706104f4659ba602c840ae